spark/v3.0/spark-extensions/src/test/java/org/apache/iceberg/spark/extensions/TestMigrateTableProcedure.java

```diff
@@ -84,7 +84,7 @@ public void testMigrateWithOptions() throws IOException {
     sql("INSERT INTO TABLE %s VALUES (1, 'a')", tableName);
 
     Object result =
-        scalarSql("CALL %s.system.migrate('%s', map('foo', 'bar'))", catalogName, tableName);
+        scalarSql("CALL %s.system.migrate('%s', false, map('foo', 'bar'))", catalogName, tableName);
```

Comment:
I thought it's backward compatible, why do we need to change existing tests?

Reply:
Fixed in the latest commit.